siliconcompiler/targets/asic_demo.py

```diff
@@ -0,0 +1,31 @@
+import os
+import siliconcompiler
+
+def setup(chip):
+    '''
+    "Self-test" target which builds a small 8-bit counter design as an ASIC,
+    targeting the Skywater130 PDK.
+
+    This target is not intended for general-purpose use.
+    It is intended to quickly verify that SiliconCompiler is installed and configured correctly.
+    '''
+
+    # Load the Sky130 PDK/standard cell library target.
+    design = 'heartbeat'
+    chip.load_target('skywater130_demo')
```

Comment:
This makes perfect sense to me so there's no actionable item for this PR, but loading a target-within-a-target does cause ['option', 'target'] to get overwritten with the last one. I just wanted to flag that I opened a ticket to track how we want to handle that, for future discussion (SC-194).

Reply:
Yeah, it's a little bit of a workaround, but that ends up working out in this case because the sky130 target is what we want for the build.